tests/rbd_mirror/test_non_default_to_non_default_scale.py

```diff
@@ -0,0 +1,304 @@
+                        f"SUCCESS: Namespace {namespace_name} got created in pool {pool} "
+                    )
+                else:
+                    log.error(
+                        f"FAIL: Namespace {namespace_name} creation failed in pool {pool} \
+                        with error {ns_create_err}"
+                    )
+                    return 1
+                # Create remote-namespace on Secondary cluster
+                remote_namespace_name = "Namespace_remote" + str(i)
+                ns_create_kw = {}
+                ns_create_kw["namespace"] = remote_namespace_name
+                ns_create_kw["pool-name"] = pool
+                (_, ns_create_err) = rbd_secondary.namespace.create(**ns_create_kw)
+                if not ns_create_err:
+                    log.info(
+                        f"SUCCESS: Namespace {remote_namespace_name} got created in pool {pool}"
+                    )
+                else:
+                    log.error(
+                        f"FAIL: Namespace {remote_namespace_name} creation failed in pool {pool} \
+                        with error {ns_create_err}"
+                    )
+                    return 1
+
+                # Enable Namespace level mirroring
+                namespace_config = {
+                    "mode": kw.get("config", {}).get(pool_type, {}).get("mode"),
+                    "namespace": namespace_name,
+                    "remote_namespace": remote_namespace_name,
+                }
+                enable_namespace_mirroring(
+                    rbd_primary, rbd_secondary, pool, **namespace_config
+                )
+
+                # Create One image in each namespace
+                image_size = kw.get("config", {}).get(pool_type, {}).get("size")
+                image_name = "image" + str(i)
+                new_image_spec = f"{pool}/{namespace_name}/{image_name}"
+                (image_create_status, err) = rbd_primary.create(
+                    **{"image-spec": new_image_spec, "size": image_size}
+                )
+                if err:
+                    raise Exception(
+                        "Failed in creating image: " + new_image_spec + ", err: " + err
+                    )
+                log.info(
+                    "Successfully created image for mirroring: "
+                    + str(image_create_status)
+                )
+
+                fio = kw.get("config", {}).get("fio", {})
+                scale(
+                    pool,
+                    namespace_name,
+                    remote_namespace_name,
+                    image_name,
+                    fio,
+                    client_primary,
+                    pri_config,
+                    sec_config,
+                    rbd_secondary=rbd_secondary,
+                    rbd_primary=rbd_primary,
+                )
+
+    log.info("Successfully passed Scale for namespace-level mirrored images")
+
+    return 0
+
+
+def scale(
```

Comment:
We could place this method in the `workflow` or `utils` folder so it can be reused going forward.